packages/web-components-stencil/src/components/link/link.tsx

```diff
@@ -0,0 +1,67 @@
+import { Component, Element, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'lux-link',
+  styleUrl: 'link.scss',
+  shadow: true,
+})
+export class Link {
+  @Element() hostElement!: HTMLElement;
+  /**
+   * Laat de browser de {@link href} als download aanbieden. Geef een string op om een bestandsnaam te suggereren
+   *
+   * @member string
+   * @access public
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#download
+   */
+  @Prop() public readonly download?: string;
+
+  /**
+   * @access public
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#href
+   */
+  @Prop() public readonly href!: string;
+
+  /**
+   * @access public
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target
+   */
+  @Prop() public readonly target?: HTMLAnchorElement['target'];
+
+  @Prop() forceState?: 'active' | 'focus' | 'hover';
```

Comment:
```suggestion
  /**
   * Force the link to (visually) be in a specific user state.
   * @access private
   */
  @Prop() forceState?: 'active' | 'focus' | 'hover';
```

Is het handig om zoiets toe te voegen? Dan maken we expliciet duidelijk wat het idee ervan is.